lib/interfaces/heroku_interface.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../http_post_interface'
+Dir['./lib/slack_commands/*.rb'].each { |file| require file }
+
+class HerokuInterface < HttpPostInterface
+  def handle_heroku_webhook(payload)
+    params = Rack::Utils.parse_nested_query(payload)
+    user = params['user'].split('@')[0]
+    release = "#{params['release']} " if params['release']
+    server = params['url'].split('/').last.split('.').first[8..-1]
+
+    deploys.complete_deploy(server)
+
+    SlackCommand
+      .new
+      .send_to_slack("#{user} just finished deploying #{release}to #{server}.")
```

Comment:
That's fine for now.  If it turns out we need to maintain some state for the slack api functions (like the `@heroku` field in `HerokuApi`), we can convert it to instance methods then.  I don't think there's any such state at the moment, so class methods should be fine.  :)